akka-docs/src/main/paradox/cluster-client.md

```diff
@@ -1,18 +1,18 @@
 # Cluster Client
 
 An actor system that is not part of the cluster can communicate with actors
-somewhere in the cluster via this @scala[@scaladoc[`ClusterClient`](akka.cluster.client.ClusterClient)]@java[@javadoc[`ClusterClient`](akka.cluster.client.ClusterClient)]. The client can of course be part of
+somewhere in the cluster via this @unidoc[ClusterClient]. The client can of course be part of
 another cluster. It only needs to know the location of one (or more) nodes to use as initial
-contact points. It will establish a connection to a @scala[@scaladoc[`ClusterReceptionist`](akka.cluster.client.ClusterReceptionist)]@java[@javadoc[`ClusterReceptionist`](akka.cluster.client.ClusterReceptionist)] somewhere in
+contact points. It will establish a connection to a @unidocRef[ClusterReceptionist](akka.cluster.client.ClusterReceptionist) somewhere in
```

Comment:
Using `@unidocRef` instead of `@unidoc`, because `@unidoc` gave the following error:

>  Error writing content for page cluster-client.html: 2 matches found for ClusterReceptionist, but not javadsl/scaladsl: akka.cluster.client.ClusterReceptionist, akka.cluster.typed.internal.receptionist.ClusterReceptionist

Reply:
I agree being able to point to a particular class when unidoc can't figure it out on its own is useful. Could we even do that in one directive, so allowing `@unidoc[ClusterReceptionist](akka.cluster.client.ClusterReceptionist)` or perhaps even `@unidoc[client.ClusterReceptionist]`?

Reply:
I think handling both `@unidoc[ClassName]` and `@unidoc[ClassName](path.to.ClassName)` is difficult, as we need to 

1. extract if `(path.to.ClassName)` part exists,
2. and if 1 is not found, fall back to the current logic for `@unidoc[ClassName]`

The problem with 1 is that we need the same logic as `SourceDirective`'s [resolvedSource](https://github.com/lightbend/paradox/blob/58cf5d419a69c05de25aeb82684a99fa537fe84b/core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala#L87), which is `sealed` inside Paradox.

For `@unidoc[client.ClusterReceptionist]`, I can give it a try. Hopefully that does not clutter the code to extract the `label` and filtering logic [here](https://github.com/richardimaoka/akka/blob/ad9843d32135940891999ee077d9adc1a983ddc2/project/ParadoxSupport.scala#L42L45)..

Reply:
`@unidoc[client.ClusterReceptionist]` was actually almost already handled. So I only made a slight change. 

You can see the generated page [here](https://doc.akka.io/docs/akka/current/cluster-client.html#cluster-client)